source/mainnet/net/references/multi-sig.rst

```diff
@@ -0,0 +1,448 @@
+each credential has one or several public-private key pair(s).
+
+.. note::
+
+   A public-private key pair are related keys used for asymmetric cryptography.
+   The private key is used to sign/authorize an action related to the account. NEVER share your private key with anyone.
+   In contrast, the public key is shared with other entities to verify that a signature is generated
+   by the associated private key.
+
+Each account has an ``AccountThreshold`` (number of credentials needed to sign the transaction
+initiated by that account) and one or several ``SignatureThresholds`` (number of signatures needed for a specific
+credential so that this credential is considered to have signed the transaction initiated by that
+account).
+
+The ``concordium-client`` tool (version 6.3.0 or above) facilitates creating, signing, and sending of multi-sig
+transactions for multi-sig accounts.
+This page will go through the main steps as follows:
+
+-  Create a multi-sig account by adding keys via an on-chain transaction to an account.
+-  Configure the ``concordium-client`` tool to use a multi-sig account.
+-  Create a multi-sig transaction with the ``concordium-client`` tool.
+-  Add an additional signature to a multi-sig transaction with the ``concordium-client`` tool.
+-  Send a multi-sig transaction with the ``concordium-client`` tool on-chain.
+
+Create a multi-sig account
+==========================
+
+When you create an account in one of the Concordium wallets, it is a basic account by default
+that has exactly one credential and exactly one public-private key pair for that credential.
+A default account can be converted into a multi-sig account.
+
+.. note::
+
+   The desktop wallet is the only wallet currently that has its independent flow of creating a multi-sig
+   account via its front end and generating multi-sig transactions. The two guides
+   :ref:`multi-sig accounts<overview-shared-accounts>` and :ref:`multi-sig transactions<create-multisig>` will
+   explain the flows for the desktop wallet.
+   The multi-sig transaction file, as generated by the desktop wallet, is currently NOT compatible with
+   the file generated by the ``concordium-client`` tool. You need to add additional signatures and send
+   the transaction on-chain with the wallet/tool the file was initially generated with. The intention is to use
+   the file format of the ``concordium-client`` tool in all Concordium tooling related to multi-sig signing
+   in the future and to replace the desktop wallet's file format.
+
+To add additional keys to an existing account on Concordium using the ``concordium-client`` tool, you can either:
+
+.. dropdown:: Add an additional public-private key pair to your current credential
+
+   - Step 1: Create an additional Ed25519 public-private key pair as you feel safe.
+
+   For testing purposes we recommend, the `front-end <https://cyphr.me/ed25519_tool/ed.html>`_ to generate an un-safe
+   additional key pair.
```

Comment:
Would it make sense to recommend a safe way to do this is as well, or do we want to avoid that for legal reasons?

Reply:
I think openSSH `ssh-keygen` would be a safe bet.

```sh
ssh-keygen -t ed25519
```

This is at least the approach recommended by github and others for generating ssh keys. Do you think this would work?

Reply:
Added it.